### ISSUE TITLE
brew update has inconsistent enthusiasm

### ISSUE THREAD

**Issue by ilovezfs** [Homebrew/brew#319]
```
Josephs-MacBook-Pro:Formula joe$ brew update
Already up-to-date!
Already up-to-date.
```

**Comment by UniqMartin** [comment]
The first output is probably coming from Git as there's no such string in the Homebrew code. Do you have `HOMEBREW_VERBOSE=1` in your environment?

**Comment by ilovezfs** [comment]
Nope, no HOMEBREW_VERBOSE.

**Comment by ilovezfs** [comment]
```
Josephs-MacBook-Pro:~ joe$ strings /usr/bin/git|grep Already
Already on '%s'
Already up-to-date.
Already up-to-date. Yeeah!
Already up-to-date!
Josephs-MacBook-Pro:~ joe$ strings /usr/local/bin/git|grep Already
Already on '%s'
Already up-to-date.
Already up-to-date. Yeeah!
Already up-to-date!
Josephs-MacBook-Pro:~ joe$ 
```
So yes, it's coming from git.

**Comment by ilovezfs** [comment]
It seems to be caused by having an untracked file.

**Comment by ilovezfs** [comment]
Also need to have HOMEBREW_DEVELOPER set.

**Comment by ilovezfs** [comment]
It's `git stash pop -q`.

**Comment by ilovezfs** [comment]
Closed via #320 2cd81e50513f96a657454031e52fa4aec773ea97
